core/tests/ledger_cleanup.rs

```diff
@@ -151,6 +151,41 @@ mod tests {
         }
     }
 
+    /// Obtains the benchmark config from the following environmental arguments:
+    ///
+    /// Basic benchmark settings:
+    /// - `BENCHMARK_SLOTS`: the number of slots in the benchmark.
+    /// - `BATCH_SIZE`: the number of slots in each write batch.
+    /// - `SHREDS_PER_SLOT`: the number of shreds in each slot.  Together with
+    ///   the `BATCH_SIZE` and `BENCHMARK_SLOTS`, it means:
+    ///    - the number of shreds in one write batch is `BATCH_SIZE` * `SHREDS_PER_SLOT`.
+    ///    - the total number of batches is `BENCHMARK_SLOTS` / `BATCH_SIZE`.
+    ///    - the total number of shreds is `BENCHMARK_SLOTS` * `SHREDS_PER_SLOT`.
+    /// - `NUM_WRITERS`: controls the number of concurrent threads performing
+    ///   shred insertion.  Default: 1.
+    ///
+    /// Advanced benchmark settings:
+    /// - `STOP_SIZE_BYTES`: if specified, the benchmark will count how
+    ///   many times the ledger store size exceeds the specified threshold.
+    /// - `STOP_SIZE_ITERATIONS`: when `STOP_SIZE_BYTES` is specified, the
+    ///   benchmark will stop immediately when the number of times where the
+    ///   ledger store size exceeds the configured `STOP_SIZE_BYTES`.  These
+    ///   configs are used to make sure the benchmark runs successfully under
+    ///   the storage limitation.
+    /// - `CLEANUP_BLOCKSTORE`: if true, the ledger store created in the current
+    ///   benchmark run will be deleted.  Default: true.
+    /// - `NO_COMPACTION`: whether to stop the rocksdb's background compaction
+    ///   completely.  Default: false.
+    /// - `ASSERT_COMPACTION`: if true, any RocksDB compaction will hit an
+    ///   assertion.  Default: false.
```

Comment:
nits:
- Consistency on `rocksdb` vs. `RocksDB`
- Do you think these two settings should be moved under the `Cleanup-service related settings` section too?
- `ASSERT_COMPACITON` can't explicitly be tied to compaction by Rocks or LedgerCleanupService; rather, the assertion simply checks if the storage size shrunk

Reply:
Thanks for the correction.  Will correct the comment to reflect that the code does.

Reply:
> Do you think these two settings should be moved under the Cleanup-service related settings section too?

I think `ASSERT_COMPACTION` should go under the cleanup service, while NO_COMPACTION should be independent of the cleanup service, as we can run a benchmark with neither cleanup service nor compaction to see how fast we can do without them.